pkg/agent/npl/k8s/annotations.go

```diff
@@ -0,0 +1,165 @@
+// +build !windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8s
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/vmware-tanzu/antrea/pkg/util/env"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+)
+
+const NPLAnnotationStr = "npl.antrea.io"
+
+type NPLEPAnnotation struct {
+	PodPort  string `json:"podPort"`
+	NodeIP   string `json:"nodeIP"`
+	NodePort string `json:"nodePort"`
+}
+
+func stringify(serialize interface{}) string {
+	jsonMarshalled, _ := json.Marshal(serialize)
+	return string(jsonMarshalled)
+}
+
+func isNodePortInAnnotation(s []NPLEPAnnotation, nodeport string) bool {
+	for _, i := range s {
+		if i.NodePort == nodeport {
+			return true
+		}
+	}
+	return false
+}
+
+func assignPodAnnotation(pod *corev1.Pod, containerPort, nodeIP, nodePort string) {
+	var err error
+	current := pod.Annotations
+	if current == nil {
+		current = make(map[string]string)
+	}
+
+	klog.V(0).Infof("Building annotation for Pod: %s\tport: %s --> %s:%s", pod.Name, containerPort, nodeIP, nodePort)
```

Comment:
`klog.Infof` equals to `klog.V(0).Infof`, I meant these logs seem too verbose to be dumped by default. I see almost all logs introduced in this PR are at least in `V(0)`, not sure if it cause logs rotated too quickly. Could we make some detailed logs `V(2)` or above?